src/features/lightspeed/vue/views/contentCreatorHelper.ts

```diff
@@ -0,0 +1,329 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+
```

Comment:
I think it would make sense to continue to use one single helper.ts to handle the event/callback, this way it will become easier to encourage reusability between webviews. However, to improve maintainability, it would be nice to maintain the logic in dedicated modules/classes.